go-controller/pkg/ovn/ovn.go

```diff
@@ -93,23 +93,32 @@ const (
 func dialNB() (*ovsdb.OVSDB, *dbcache.Cache) {
 	var addrList [][]string
 
-	addr := strings.SplitN(config.OvnNorth.ClientAuth.OvnAddressForClient, ":", 2)
+	addresses := strings.Split(config.OvnNorth.ClientAuth.OvnAddressForClient,
+		",")
 
 	var options map[string]interface{}
+	var sslUse bool
```

Comment:
I would call it useSSL, as sslUse feels like a definition how the SSL is used

Reply:
Done